elisp-bytecode.texi

```diff
@@ -1344,164 +1818,164 @@ Make a binding recording buffer, point, and mark.
 
 @item @verb{|  00|}
 @tab @verb{|   0|}
-@tab @tab Not used; see @code{dup}
+@tab @tab An error. Before 25.1 it is an immediate program abort! Logically @code{stack-ref[0]} but @code{dup} should be used instead.
 
 @item @verb{|  01|}
 @tab @verb{|   1|}
-@tab @verb{|  stack-ref1|}
+@tab @verb{|  stack-ref[1]|}
 @tab stack reference 1
 @tab @math{+1}
 @item @verb{|  02|}
 @tab @verb{|   2|}
-@tab @verb{|  stack-ref2|}
+@tab @verb{|  stack-ref[2]|}
 @tab stack reference 2
 @tab @math{+1}
 @item @verb{|  03|}
 @tab @verb{|   3|}
-@tab @verb{|  stack-ref3|}
+@tab @verb{|  stack-ref[3]|}
 @tab stack reference 3
 @tab @math{+1}
 @item @verb{|  04|}
 @tab @verb{|   4|}
-@tab @verb{|  stack-ref4|}
+@tab @verb{|  stack-ref[4]|}
 @tab stack reference 4
 @tab @math{+1}
 @item @verb{|  05|}
 @tab @verb{|   5|}
-@tab @verb{|  stack-ref5|}
+@tab @verb{|  stack-ref[5]|}
 @tab stack reference 5
 @tab @math{+1}
 @item @verb{|  06|}
 @tab @verb{|   6|}
-@tab @verb{|  stack-ref6|}
+@tab @verb{|  stack-ref[6]|}
```

Comment:
I think I messed up disassembling this instruction correctly, but I'll have to check. Should it be, say, `stack-ref[6] [255]`